src/App.js

```diff
@@ -1,12 +1,29 @@
 import "./App.css";
 import Form from "./components/Form/Form";
+import List from "./components/List/List";
 import React from "react";
 import { useState } from "react";
+import useLocalStorageState from "use-local-storage-state";
 
 function App() {
-  const [activities, setActivities] = useState([]);
+  // const [activities, setActivities] = useState([]);
 
-  function handleAddActivity(newActivity) {
+  const isForGoodWeather = false;
+
+  const [activities, setActivities] = useLocalStorageState("activities", {
+    defaultValue: [],
+  });
```

Comment:
It's recommended to place the state at the top - in some cases this is even a technical requirement. But it is also recommended for the overview and readability.

```suggestion
  const [activities, setActivities] = useLocalStorageState("activities", {
    defaultValue: [],
  });

  const isForGoodWeather = false;


```